src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -1572,34 +1572,34 @@ def input(
 
         Parameters
         ----------
-        fname : str
+        fname : str, optional
             MAPDL input file to stream to the MAPDL grpc server.
```

Comment:
```suggestion
            MAPDL input file to stream to the MAPDL gRPC server.
```
@clatapie Rather than continue nitpicking in this PR, I should do an overall doc edit. Last I asked about this, Alex said to wait until the work with breaking out examples had been completed. Is it now OK to do PRs for doc edits of different sections?

Reply:
Indeed, it might be easier. As I was not aware about this guideline, I'll let @germa89 answer that question.

Reply:
@PipKat feel free to do those PRs. The breaking out of the examples is completed.